estate/models/estate_property.py

```diff
@@ -8,18 +8,33 @@ class EstateProperty(models.Model):
     name = fields.Char(required=True)
     description = fields.Text()
     postcode = fields.Char()
-    date_availability = fields.Date()
+    date_availability = fields.Date(copy=False, default=lambda self: fields.Date.add(fields.Date.today(), months=3))
     expected_price = fields.Float(required=True)
-    selling_price = fields.Float()
-    bedrooms = fields.Integer()
+    selling_price = fields.Float(readonly=True, copy=False)
```

Comment:
Attributes should be organized alphabetically.

```suggestion
    selling_price = fields.Float(copy=False, readonly=True)
```